partials/forms/checkbox-group.html

```diff
@@ -0,0 +1,26 @@
+<div id="{{key}}-group" class="form-group{{#className}} {{className}} {{/className}}{{#error}} error{{/error}}">
+    <fieldset role="group" aria-required="true">
+        <legend>
+            <span{{#legendClassName}} class="{{legendClassName}}"{{/legendClassName}}>{{legend}}</span>
+            {{#hint}}<span id="{{key}}-hint" class="form-hint">{{hint}}</span>{{/hint}}
+            {{#error}}<span id="{{key}}-error" class="error-message">{{error.message}}</span>{{/error}}
+        </legend>
+        {{#options}}
+            <label class="block-label" for="{{key}}-{{value}}">
+                <input
+                    type="checkbox"
+                    name="{{key}}"
+                    id="{{key}}-{{value}}"
+                    value="{{value}}"
+                    required="true"
+                    {{#toggle}} data-toggle="{{toggle}}"{{/toggle}}
+                    {{#selected}} checked="checked"{{/selected}}
+                    {{^error}}{{#hint}} aria-describedby="{{key}}-hint"{{/hint}}{{/error}}
+                    {{#error}} aria-describedby="{{key}}-error" aria-invalid="true"{{/error}}
+                >
+                {{{label}}}
+            </label>
+            {{#renderChild}}{{/renderChild}}
```

Comment:
Good idea, or we could rename this `option-group.html` and use the existing option group function to pass in the field type?